bigtable/integration_test.go

```diff
@@ -2259,3 +2343,34 @@ func isDirectPathRemoteAddress(testEnv IntegrationEnv) (_ string, _ bool) {
 	// DirectPath ipv6 can use either ipv4 or ipv6 traffic.
 	return remoteIP, strings.HasPrefix(remoteIP, directPathIPV4Prefix) || strings.HasPrefix(remoteIP, directPathIPV6Prefix)
 }
+
+func blackholeOrAllowDirectPath(testEnv IntegrationEnv, t *testing.T, blackholeDP bool) {
```

Comment:
Can we split this into two functions, one corresponding to `blackholeDP=true` and one for `false`?

That would get rid of the need for the `blackholeDP` argument at all, making calls more clear and easier to debug.

Reply:
Done